crypt/test/simulation/TestGenerateSteadyStateCrypt.hpp

```diff
@@ -169,7 +169,7 @@ class TestGenerateSteadyStateCrypt : public CxxTest::TestSuite
 
             // Define some "sensible" bounds on the number
             unsigned expected_cell_count_ub = 480u;
-            unsigned expected_cell_count_lb = 425u;
+            unsigned expected_cell_count_lb = 420u;
```

Comment:
This shouldn't be changing in this commit? (Might need to merge in latest develop?)